### PR TITLE
Graph polish

### PR DESCRIPTION
**Pull request by jaredoconnell**
This PR:
- Makes it so it doesn't error out due to the model when I refresh the page
- Restructures the graph to make it so the text is visible and looks appropriately positioned
- Brings more attention to the colors and lowers vertical size of graph to de-emphasize it.

Example of the graph:
image

Known limitations: These are still fixed size values, so when the heights of the buckets in the histogram change, it throws off the sizing of things. In the future, we will need to either use a different method of annotation, or we'd need to find the maximum bucket size and make everything relative to that. There is no easy solution to get the bucket size from the graph, so it will need to be calculated separately.

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 class Run_Metrics(SQLModel, table=True):
     __tablename__ = 'run_metrics'
+    __table_args__ = {'extend_existing': True}
```
Comment (mfleader) [inline model.py:10]: why did you add this?

Comment (jaredoconnell) [reply model.py:10]: Every single time I refreshed it, it failed because Run_Metrics was already loaded. This fixes it
Screenshot from 2022-07-08 11-59-48
.

Comment (mfleader) [reply model.py:10]: awesome! I tried several changes to fix that.
